aqua/smooth/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        enzo1982 smooth 0.9.6 v
+github.tarball_from releases
+
+categories          aqua
+platforms           darwin
+license             Artistic-2
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         ${name} is a class library for user interfaces written in C++
+long_description    ${name} is an object oriented C++ class library for Windows, macOS, Linux and most Unix-like operating systems. \
+                    It provides basic functionality and platform support for applications and libraries. \
+                    \n \
+                    \nFeatures provided by smooth include:\
+                    \n* user interface API with various widgets\
+                    \n* simple to use multithreading API\
+                    \n* file and network IO interface\
+                    \n* completely transparent Unicode and software internationalization support\
+                    \n* a libxml2 based XML parser\
+homepage            http://www.smooth-project.org/
+
+depends_lib         port:git \
+                    port:bzip2 \
+                    port:curl \
+                    port:fribidi \
+                    port:gtk3 \
+                    port:jpeg \
+                    port:libxml2
```

Comment:
Please sort these alphabetically

Reply:
Done. I also removed git as it is not a dependency. I added this in the first place because upstream is a github project, but now I realized that I dont use git to fetch the sources.

Reply:
Done. I also removed git because it turned out that it is not a dependency.